Readability.js

```diff
@@ -577,10 +577,12 @@ Readability.prototype = {
     this._clean(articleContent, "link");
     this._clean(articleContent, "aside");
 
-    // Clean out elements have "share" in their id/class combinations from final top candidates,
+    // Clean out elements with little content that have "share" in their id/class combinations from final top candidates,
     // which means we don't remove the top candidates even they have "share".
-    this._forEachNode(articleContent.children, function(topCandidate) {
-      this._cleanMatchedNodes(topCandidate, /share/);
+    this._forEachNode(articleContent.children, function (topCandidate) {
+      this._cleanMatchedNodes(topCandidate, function (node, matchString) {
+        return /share/.test(matchString) && node.textContent.length < 500;
```

Comment:
Could we reuse `DEFAULT_CHAR_THRESHOLD` here? (Note: `_removeNodes`, unlike `_cleanMatchedNodes`, keeps `this` the same when applying the filter function so we can just access `this.DEFAULT_CHAR_THRESHOLD`, I think - but worth doublechecking.)

Reply:
Done.